SPECS/systemd/systemd.spec

```diff
@@ -12,7 +12,10 @@ Source1:        50-security-hardening.conf
 Source2:        systemd.cfg
 Source3:        99-dhcp-en.network
 Patch0:         fix-journald-audit-logging.patch
+# Patch for skipping the tests: test-mountpoint-util, test-mount-util, test-fileio, test-fd-util, test-repart - Some mounts are failing in chroot
+Patch1:         testsskipped.patch
```

Comment:
Could we modify the tests in such way, that they will be ignored or passing in the chroot but start failing if we ever run them bare metal or in a VM? I wouldn't want to keep the tests ignored, if we ever get to an environment where they should actually pass.

Reply:
Sure, I'll see if I can modify them that way

Reply:
The changes don't look trivial, I'll investigate further and update the patch in a future PR.